src/sentry/api/serializers/models/project.py

```diff
@@ -159,16 +159,31 @@ def get_attrs(self, item_list, user):
         attrs = super(ProjectWithTeamSerializer,
                       self).get_attrs(item_list, user)
 
+        project_teams = list(
+            ProjectTeam.objects.filter(
+                project__in=item_list,
+            ).select_related('team')
+        )
+
+        # TODO(jess): decide if we should only include teams the user
```

Comment:
gonna say it's ok to include teams a user doesn't have access to since we allow users to see team names and request access to teams they don't have access to

Reply:
Yeah, I think that's reasonable. Teams aren't private, at least knowing they exist. Only the contents within.